doc/getting-started.md

```diff
@@ -33,6 +33,19 @@ storage has priority over the one you're going to use.
 > You may need to erase it, or do other manipulations on your board to skip it
 > for a single boot.
 
+After writing the firmware, the installer will report success, and depending,
+either reboot on any key input, or allow you to go back to the main menu, for
+the graphical installer.
+
```

Comment:
We shouldn't get too hung-up on that, and instead consider (for later) directing the user to the device-specific page, which will know which installer the platform use, and correctly describe the installation flow.